### PR TITLE
docs: Add LICENSE and update package.json files

### PR DESCRIPTION
**Pull request by pmareke**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ guidelines of ESLint to develop a custom plugin, but it includes also style form
 [prettier](https://prettier.io/) and [standard-version](https://github.com/conventional-changelog/standard-version) 
 * [@empathyco/x-logger](https://github.com/empathyco/x/tree/main/packages/x-logger) - A custom implementation of default node's logger.
 * [@empathyco/x-get-safe-property-chain](https://github.com/empathyco/x/tree/main/packages/x-get-safe-property-chain) - A utility for safely retrieving nested values.
+* [@empathyco/x-deep-merge](https://github.com/empathyco/x/tree/main/packages/x-deep-merge) - Clones an array of objects into another object.
```
Comment (pmareke) [inline .github/CONTRIBUTING.md:78]: https://github.com/empathyco/x/tree/main/packages/eb-deep-merge

**Review by pmareke — COMMENTED** [review]
Change url, it's not okay.

File: packages/eb-deep-merge/package.json [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {
+  "author": "Empathy Systems Corporation S.L.",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/empathyco/x/tree/main/packages/x-deep-merge#readme",
   "name": "@empathyco/x-deep-merge",
   "version": "1.2.1",
   "description": "Clones an array of objects into another object.",
```
Comment (LuisMartinez15) [inline packages/eb-deep-merge/package.json:7]: I'd change this description to `Utility for deep cloning objects.`

```suggestion
  "description": "Utility for deep cloning objects.",
```

**Review by LuisMartinez15 — CHANGES_REQUESTED** [review]
I'd also update the `README.md` content and change it a little bit. Here you have a suggestion:

# x-deep-merge

Utility for deep cloning objects.

### How to install

```
npm install @empathyco/x-deep-merge --save
```

### How to use

[TypeScript Module Resolution Logic](https://www.typescriptlang.org/docs/handbook/module-resolution.html) makes it quite easy. The `package.json` file contains `main` attribute that points to the generated `dist/client.js` file and `typings` attribute that points to the generated `dist/client.d.ts` file. You can also import this package as a module:

- Using `x-deep-merge` in a `TypeScript` file:

```ts
import { deepMerge } from '@empathyco/x-deep-merge';

const target = {};
const source1 = { a: 1 };
const source2 = { b: 2 };

deepMerge(target, source1, source2); // target = { a: 1, b: 2 }
```

- Another example

```ts
import { deepMerge } from '@empathyco/x-deep-merge';

const sources = [{ a: 1 }, { b: 2 }];
const mergedObject = deepMerge({}, ...sources); // mergedObject = { a: 1, b: 2 }
```

If you want to find out all the covered use cases, check the `tests` folder.

### How to update the version

You can check if a new version has been published running [npm outdated](https://docs.npmjs.com/cli/outdated):

`npm outdated @empathyco/x-deep-merge`

And update it using [npm update](https://docs.npmjs.com/cli/update):

`npm update --save @empathyco/x-deep-merge`

**Review by LuisMartinez15 — APPROVED** [review]

File: packages/deep-merge/package.json [hunk 3]
```diff
@@ -21,7 +24,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://bitbucket.org/colbenson/eb-deep-merge.git"
+    "url": "git+https://github.com/empathyco/x.git"
```
Comment (LuisMartinez15) [inline packages/deep-merge/package.json:27]: This should be the url to the repo, without `git+ssh` or `git+https` as specify in the [documentation examples](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository).

**Review by LuisMartinez15 — CHANGES_REQUESTED** [review]

File: packages/deep-merge/package.json (hunk 3 above)
Comment (pmareke) [reply packages/deep-merge/package.json:27]: I'll fix this `repository` property in the last PR.

**Review by javieri-empathy — APPROVED** [review]

**Review by LuisMartinez15 — APPROVED** [review]

**Pull request merged**